plugins/woocommerce/src/Internal/Admin/Reviews.php

```diff
@@ -10,4 +10,97 @@
  */
 class Reviews {
 
+	/**
+	 * Class instance.
+	 *
+	 * @var Reviews|null instance
+	 */
+	protected static $instance;
+
+	/**
+	 * Reviews page hook name.
+	 *
+	 * @var string
+	 */
+	protected $reviews_page_hook;
```

Comment:
Maybe a constant would make more sense here?

Reply:
I don't think so: this is dynamically populated by a method below

I would change the approach and hardcode it if we need the same hook in other classes